utils.js

```diff
@@ -132,13 +133,17 @@ module.exports = {
 
   /**
    * Simplify superagent's http response object
-   * @param {object} r
+   * @param {object} r - The response.
+   * @param {object} Request - The superagent's Request instance.
+   * @param {object} props  - The request properties.
    */
-  gutResponse: function(r){
+  gutResponse: function(r, Request, props){
```

Comment:
Could go for it, the name comes from the original author though ;) Didn't want to correct everything as the PR to original upstream is already huge enough to let the author work a bit on it ;)